packages/circuit-ui/components/NotificationToast/NotificationToast.docs.mdx

```diff
@@ -0,0 +1,121 @@
+
+Use the 'Confirm' variant the successful messages.
+
+### Notify
+
+Use the 'Notify' variant for warning messages, and other information a user should be aware of.
+
+### Alert
+
+Use the 'Alert' variant for error messages.
+
+## Usage in code
+
+First, wrap your application in the `ToastProvider` which keeps track of the open toasts and ensures the accessibility of the toast.
+
+```tsx
+// _app.tsx for Next.js or App.js for CRA
+import { ToastProvider } from '@sumup/circuit-ui';
+
+export default function App() {
+  return <ToastProvider>{/* Your content goes here... */}</ToastProvider>;
+}
+```
+
+Then, use the `useNotificationToast` hook to open a toast from a component:
+
+```tsx
+import { useNotificationToast, Button } from '@sumup/circuit-ui';
+
+export function App({}) {
+  const { setToast } = useNotificationToast();
+
+  const handleClick = () => {
+    setToast({
+      body: 'This is a toast message',
+    });
+  };
+
+  return <Button onClick={handleClick}>Open toast</Button>;
+}
+```
+
+## Accessibility
+
+By their nature as small quick messages, or interruptions, toasts map to [live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).
+Live region must always exist on the page at render, before the toast is generated.
+
+Ideally, the toast messages should not contain any interactive elements.
+The screen readers are gonna announce the raw text within toast messages, without structure and any need to move the user's focus.
+Keyboard users won’t be able to move to the interactive elements with a clear path.
+For example, there is a toast message with the text “You successfully updated your data.”, followed by a link or button with the text "Back".
+Screen readers are gonna communicate a message as “You successfully updated your data. Back.”, and the semantics of the link "Back" is not announced.
+
+To match the content, the role and aria-live level is adapted. The role is set to `"status"`, and it has `aria-live="polite"` meaning the screen reader will wait before announcing the update.
+
+### Best practices
+
+This component adheres to the following principles from the Web Content Accessibility Guidelines (WCAG):
+
+#### Toast duration
+
+The auto-dimiss duration of the toast is adjustable.
+Based on how fast the average American reads, a good lenght of time to keep the toast message up is 5 seconds plus 1 extra second for every 120 words.
+The shortest default that should be used as a best practice is 6 seconds.
+
+#### Consistent identification
+
+Based on the WCAG guideline [3.2.4](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html) each component should be consistent with another.
+
+#### Location
+
+Following consistent identificaton, toast messages should be also placed consistently. Generally toast messages usually appear at the start or end of the DOM. If you place them somewhere else, it can cause difficulties for screen reader users to locate them.
```

Comment:
This seems wrong—screen reader users probably wouldn't have time to reach the toast in the DOM before it disappears anyways, that's why we're using a live region. I guess that a consistent location at the bottom center of the screen is useful for sighted users, though